library/pkparse.c

```diff
@@ -74,6 +80,10 @@ int mbedtls_pk_load_file( const char *path, unsigned char **buf, size_t *n )
     FILE *f;
     long size;
 
+    PK_VALIDATE_RET( path != NULL );
+    PK_VALIDATE_RET( buf != NULL );
```

Comment:
Would additionally checking `*buf` be within the current design philosophy of the check params feature?

Reply:
No, because `*buf` is being set by this function; its initial state doesn't matter.

Reply:
Yup, agreed.